tmt/schemas/provision/hardware.yaml

```diff
@@ -203,6 +206,9 @@ definitions:
       memory:
         "$ref": "#/definitions/memory"
 
+      model:
```

Comment:
Must be listed also in `and`, `or` and `hardware` objects below.

Reply:
@happz thank you for the review. I will push updates soon.

btw, I noticed "memory" is not in listed in "and", "or" and "hardware" objects.
I can add it as well. Please let me know

Reply:
Ouch, yes, that might be a bug. If you're willing to fix the issue, please, feel free to do so in a separate patch.